map_frame: map
local_frame: local
body_frame: body
imu_topic: /imu/data_raw
rs_m1_topic: /os1_points

local_rate: 20.0
loop_rate: 1.0

# 是否pub地图点云
publish_map_cloud: true

lio_builder:
  det_range: 100.0
  cube_len: 500.0
  resolution: 0.2
  move_thresh: 1.5
  align_gravity: true
  imu_ext_rot: [1, 0, 0, 0, -1, 0, 0, 0, -1]
  imu_ext_pos: [0.065, 0.025, -0.18]

localizer:
  refine_resolution: 0.2
  rough_resolution: 0.5
  refine_iter: 20
  rough_iter: 50
  thresh: 0.12
  xy_offset: 2.0
  yaw_offset: 5
  yaw_resolution: 2.0